app/helpers/spotlight/application_helper.rb

```diff
@@ -136,16 +136,35 @@ def available_view_fields
     end
 
     def exhibit_stylesheet_link_tag(tag)
-      if current_exhibit_theme && current_exhibit.theme != 'default'
```

Comment:
This seems to me to be another backwards compatibility issue.

Current behavior:

If no theme is configured, then it uses `application.css` for the default theme

New behavior:

If no theme is configured, tries to use `application_default.css`

Reply:
That shouldn't be happening.  This test is supposed to ensure that it's still `application.css`: https://github.com/projectblacklight/spotlight/pull/2151/files#diff-e1e92e3b842cf6d2c8b95afc21c06044R71

How can I reproduce what you're seeing?

Reply:
Here are the steps I'm following:

1. `rm -rf .internal_test_app`
1. Recreate a new spotlight app
1. Configure two themes 

```ruby
# .internal_test_app/config/initializers/spotlight_initializer.rb
Spotlight::Engine.config.exhibit_themes = ['default', 'fancy']
```

Then after starting up application and choosing a site theme as "default", no css is found to render that.

![Screen Shot 2019-04-16 at 12 53 24 PM](https://user-images.githubusercontent.com/1656824/56236354-a395ee00-6046-11e9-9237-9b6c687f288b.png)

Reply:
One thing I did that probably just ended up being confusing was to make `default` be the name of a theme; it's doesn't mean that no theme is used.  I did that because (as far as I can see), once you select a theme there's no way to revert to using no theme.  So I added a reference to a fallback 'default' theme as a workaround.

I can start working on the root problem (letting admins un-theme exhibits), but I'd rather that be a subsequent PR.